packages/graphql/common.js

```diff
@@ -0,0 +1,41 @@
+require('isomorphic-fetch');
+
+var React = require('react');
+var ReactApollo = require('react-apollo');
+
+var Context = require('hops-redux').Context;
+
+exports.Context = exports.createContext = Context.extend({
+  initialize: function (options) {
+    Context.prototype.initialize.call(this, options);
+    this.client = this.createClient(Object.assign(
+      { network: {} },
+      options.graphql
+    ));
+    this.registerReducer('apollo', this.client.reducer());
+  },
+  createClient: function (options) {
+    return new ReactApollo.ApolloClient(
+      Object.assign(
+        options,
+        {
+          networkInterface: ReactApollo.createNetworkInterface(options.network)
```

Comment:
In order to use a fully mocked network interface, I'd propose to use s.th. like: 

```
networkInterface: options.networkInterface || ReactApollo.createNetworkInterface(options.network)
```

A mocked networkInterface could look like this:

```js
import { makeExecutableSchema, addMockFunctionsToSchema } from 'graphql-tools';

const typeDefs = `
type Foo {
  bar: String
}
type Query {
  foo: Foo
}
```;
const schema = makeExecutableSchema({ typeDefs });
addMockFunctionsToSchema({
  schema,
  mocks: {
    Foo() {
      return {
        bar: 'baz'
      };
    }
  }
});
const networkInterface: {
  query (request) {
    return graphql(
      schema,
      print(request.query),
      {},
      {},
      request.variables,
      request.operationName
    );
  }
}
```